### ISSUE TITLE
Determine language/copy for viewer config page

### ISSUE THREAD

**Issue by csbailey5t** [sul-dlss/exhibits#481]
See this branch: https://github.com/sul-dlss/exhibits/tree/add_mirador_as_viewer

The feature (but not tests) to add Mirador as a viewer instead of the standard SUL-Embed is added. Following the template of the bibliography service, we add a menu option for the exhibit to get to a page to choose between Mirador and SUL-Embed with a radio button. We need to clarify the text for the exhibit administrator. 

Here's the current state: 
<img width="1114" alt="screenshot 2017-05-19 09 06 18" src="https://cloud.githubusercontent.com/assets/4672869/26256625/77dd28c2-3c72-11e7-99e9-d092db1b7109.png">

**Comment by ggeisler** [comment]
@csbailey5t: This looks good. I have some minor suggestions below. Feel free to override anything I suggest that doesn't make sense to you.

![configuration_-_viewers___mirador_feature_text_-_stanford_university_libraries](https://cloud.githubusercontent.com/assets/101482/26271618/2a29d03a-3cbd-11e7-83b7-e84f2e4aff6d.png)

Here's what I've updated:

* I changed the section title to "Item Detail Page" because having "Viewers" as a heading twice is not ideal and if I understand correctly, this configuration is specific to the Item Detail page (and not, for example, the Embed widget), so it doesn't apply to all image resources displayed in the app. Plus, if we ever have more viewer configurations we can use tabs to separate them and turn that heading into the tab heading.

* Minor rewording of the instructions text:

> By default, the Item Detail page displays image resources using SUL-Embed as the viewer. You can change this setting to use a pre-configured version of Mirador as the viewer instead. The Mirador viewer provides minimal options (similar to 'zen mode'), but does include annotation viewing for any IIIF manifests with annotations.

* I capitalized the "embed" part of "SUL-Embed" for the radio button choice, just to be consistent with how we capitalized it in the instructions text.

Also, your screenshot above doesn't reflect this, but when I ran the branch locally neither radio button was selected. We definitely want to show the SUL-Embed button as the default selected choice, if it isn't already.
